shape/RectByPoints.java

```diff
@@ -4,10 +4,7 @@
    private Rect rect; // This is called Delegation
 
    public RectByPoints(double[] a, double[] b) {
-	  if (a.length==2 && b.length==2)
 		  this.rect = new Rect( Math.abs(a[1]-a[0]),  Math.abs(b[1]-b[0])); // this should be implemented for real some time :)
```

Comment:
Im pretty sure this is not how it works geometrically :)

Reply:
Rect is well defined with 2 points and parallel to the x-axis and y-axis.
The length of the sides is equal to the differences between x and y coordinates.
How to define it in another way? What do you have in mind?